lib/l10n/L.dart

```diff
@@ -447,6 +447,16 @@ class L {
         );
   }
 
+  String get settings_delete_played_label {
+    return message('settings_delete_played_label') ??
+        Intl.message(
+          'Delete downloaded played episodes',
```

Comment:
Great that you found the Strings settings OK as I realised now that I have no documentation on handling translations, which I will address shortly. 

I think this string would read better as `Delete downloaded episodes once played` because it is closer in language to `Mark deleted episodes as played`. I also think it this setting should be moved up one so it is directly under `Mark deleted episodes as played` and before the SD card setting.

Reply:
ok! np.